tests/mocha/astnode_test.js

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+goog.module('ASTTest');
```

Comment:
Is there a good reason not to name test modules consistently with how core modules are named?  So this could be `Blockly.test.astTest`, for example.